master/reference/advanced/etcd-rbac/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Setting up etcd certificates for RBAC
+---
+
+When using etcd it is a good idea to protect the data stored there.  This is
+even more true when you have multiple components using a common etcd cluster.
+
+## Why you might be interested in this guide
+
+- You want to restrict the operations that support staff are able to perform
+  to minimize accidental corruption or deletion of etcd data.
+- You are using the etcd cluster with multiple orchestrators (Kubernetes,
+  docker, Calico) or multiple tenants and want to prevent access between the
+  multiple tenants.
+- You want to restrict the read/write access of the various Calico components
+  as an additional safety measure.
```

Comment:
Has anyone asked for this? It makes e.g. calico-etcdv2-paths.md significantly more complex, and I'm not sure how much benefit it yields.